docker-compose-dev-local-images.yaml

```diff
@@ -3,27 +3,52 @@ version: '3'
 #Build only (put your images in here)
 services:
 #An exmaple of how to add your code to databox
-  databox-os-monitor-driver:
+  driver-os-monitor:
     build:
-        context: ./databox-os-monitor-driver
+        context: ./driver-os-monitor
         dockerfile: Dockerfile${DATABOX_ARCH}
```

Comment:
My personal preference would be to make it explicit on all architectures. More consistent to maintain going forwards. (Avoid special cases!)